std/io.zig

```diff
@@ -433,7 +438,7 @@ fn parseU64DigitTooBig() {
 
 pub fn openSelfExe(stream: &InStream) -> %void {
     switch (@compileVar("os")) {
-        linux => {
+        linux,darwin => {
```

Comment:
I don't think Darwin has a procfs - we'll need to implement this a different way for Darwin. We could for now probably just remove the `,darwin` part here since `openSelfExe` isn't (yet) covered in the test suite.